### DISCUSSION TITLE
Can't clone the repo for exercise 2

### DISCUSSION THREAD

**Discussion by benpymer** [Part-II-Computational-Physics/.github#29] [General]
Hi! I'm trying to clone the repo for exercise 2 so that I can open my assignment in VS Code, but GitHub desktop is telling me I can't because the repo contains files. I don't think I had this problem with the previous exercise (although a friend helped me set it up last time). Here's a screenshot of the message I'm getting

![Screenshot (30)](https://user-images.githubusercontent.com/125975405/221299056-4d85168b-3746-446f-9e4a-d7401617ee93.png)
 
Does anyone know what's going wrong?

Thanks!

**Comment by JNHallen** [comment]
Hi!  You need to clone the exercise to an empty folder without any pre-existing files.

**Reply by benpymer** [reply]
Thanks! This might be a really dumb question, but how do I create an empty repo? I tried doing it so that I could clone the repo to it, but it still said it contained a file called gitattributes. I deleted that file and committed the change, but it still says the repo contains files. Do I need to do something special to avoid this?

**Reply by benpymer** [reply]
I've sorted this now with the help of the friend who helped me last time
